application/common/components/notify/EmailServiceNotifier.php

```diff
@@ -85,6 +85,10 @@ public function getSiteStatus(): string
      */
     public function sendMissingEmailNotice(array $users)
     {
+        if (empty($this->emailTo)) {
+            return;
+        }
```

Comment:
Good validation add. :) But how will you tell if this function really sent or not?

Reply:
Yes, success assumed if no Exception thrown.

Reply:
But this didn't send any notice, so isn't that a failure?

Reply:
If this was new code, then I might say so. However, about half of the IDPs purposely have no email configured, which is code for "don't send these notifications". I don't want an error flagged every time this tries to send those notifications.

Reply:
Background: the notifier was only used for "missing email" notifications. Only one config variable was used to enable/disable this notification: the NOTIFIER_EMAIL_TO variable. The old code just replaced the notifier with NullNotifier when no NOTIFIER_EMAIL_TO was configured. Since I now need the notifier for two kinds of messages, I can't use NullNotifier so I need another way to block the "missing email" notification. That return is my solution for that.

Reply:
Would have been nice to reference the issue number, but sufficient. :)